addons/website/static/src/js/content/menu.js

```diff
@@ -678,12 +698,24 @@ publicWidget.registry.HeaderMainCollapse = publicWidget.Widget.extend({
      */
     _onCollapseShow() {
         this.el.classList.add('o_top_menu_collapse_shown');
+        if (this.offcanvasAndBoxedHeader) {
+            this.options.wysiwyg && this.options.wysiwyg.odooEditor.observerUnactive();
```

Comment:
I don't have the impression that `observerActive` and `observerUnactive` exist in 14.0 :thinking: When looking in the codebase, it seems that we declare a `new MutationObserver()` when needed 
Do you really need to observe ? When I remove these lines, it doesn't seem really broken ? :open_mouth: